schemas/com.snowplowanalytics.snowplow/cookie_extractor_context/jsonschema/1-0-0

```diff
@@ -0,0 +1,25 @@
+{
+	"$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+	"description": "Schema for cookie extractor context generated by cookie extractor enrichment",
+	"self": {
+		"vendor": "com.snowplowanalytics.snowplow",
+		"name": "cookie_extractor_context",
+		"format": "jsonschema",
+		"version": "1-0-0"
+	},
+
+	"type": "object",
+	"properties": {
+		"cookies": {
+			"type": "object",
+			"properties": {
+				"name": "string",
+				"value": "string"
+			},
+			"required": ["name", "value"]
+		}
+	},
+	"required": ["cookies"],
+	"additionalProperties": false
+}
```

Comment:
Is there a reason why we say browser cookies have Snowplow as their vendor?

Why not org.ietf/http_cookie or org.ietf.rfc-6265/http_cookie ?

A
On 28 Sep 2015 9:35 am, "Josh" <notifications@github.com> wrote:

> In
> schemas/com.snowplowanalytics.snowplow/cookie_extractor_context/jsonschema/1-0-0
> <https://github.com/snowplow/iglu-central/pull/215#discussion_r40528467>:
>
> > +	},
> > +
> > +	"type": "object",
> > +	"properties": {
> > +		"cookies": {
> > +			"type": "object",
> > +			"properties": {
> > +				"name": "string",
> > +				"value": "string"
> > +			},
> > +			"required": ["name", "value"]
> > +		}
> > +	},
> > +	"required": ["cookies"],
> > +	"additionalProperties": false
> > +}
>
> Looking at the test suite from the companion PR there is no "cookie"
> wrapper.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/snowplow/iglu-central/pull/215/files#r40528467>.
>